data/helpers.d/network

```diff
@@ -47,7 +47,7 @@ ynh_find_port () {
 # | arg: path - the web path to check the availability of
 ynh_webpath_available () {
 	local domain=$1
-	local path=$(ynh_normalize_url_path $2)
+	local path=$2
```

Comment:
After discussing on the chat, we agreed that :
- the normalizing of the path should be done before the install script ideally, therefore we should remove both `ynh_normalize_url_path` from these helpers
- the availability check should be done before even launching the install script

Both these points are implemented in https://github.com/YunoHost/yunohost/pull/304